modules/workflow-service-impl/src/main/java/org/opencastproject/workflow/impl/WorkflowServiceImpl.java

```diff
@@ -2387,7 +2387,13 @@ public void repopulate(final String indexName) throws Exception {
           logger.warn("Skipping restoring of workflow. Error parsing: {}", workflow, e);
           continue;
         }
-        Organization organization = instance.getOrganization();
+        Organization organization = null;
+        try {
+          organization = organizationDirectoryService.getOrganization(instance.getOrganization());
```

Comment:
... getOrganization() expects the ID of the organization. While instance.getOrganization() now indeed does return the ID, instace.getOrganizationId() would make this more obvious.